recipes/brunocodutra/metal/package.txt

```diff
@@ -0,0 +1 @@
+brunocodutra/metal@standalone -X build.cmake
```

Comment:
This can be installed with `brunocodutra/metal@master -X header`, so there's need for the extra cmake file. It just won't be a single header file, but `#include <metal.hpp>` would still work.